test/e2e/node/pods.go

```diff
@@ -179,12 +179,13 @@ var _ = SIGDescribe("Pods Extended", func() {
 		ginkgo.BeforeEach(func() {
 			podClient = f.PodClient()
 		})
+
 		/*
 			Release : v1.9
 			Testname: Pods, QOS
-			Description:  Create a Pod with CPU and Memory request and limits. Pos status MUST have QOSClass set to PodQOSGuaranteed.
+			Description:  Create a Pod with CPU and Memory request and limits. Pod status MUST have QOSClass set to PodQOSGuaranteed.
 		*/
-		framework.ConformanceIt("should be submitted and removed ", func() {
+		framework.ConformanceIt("should be set QOS Class on Pods with matching resource requests and limits for memory and cpu", func() {
```

Comment:
This is going to be even more illegible (ref: https://github.com/kubernetes/kubernetes/pull/75544#discussion_r291414867) the full test would now be "Pods Set QOS Class should be set QOS Class on Pods ..."

I was fine with the previous verbiage given the explanation I gave, and would at most suggest changing the `KubeDescribe` above from "Pods Set QOS Class" to "Pod QOS Class", which would result in "Pod QOS Class should be set on Pods..."